api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirMedicationRequestDao.java

```diff
@@ -13,8 +13,8 @@
 
 import org.openmrs.DrugOrder;
 
-public interface FhirMedicationRequestDao {
-	
-	DrugOrder getMedicationRequestByUuid(@NotNull String uuid);
+public interface FhirMedicationRequestDao extends FhirDao<DrugOrder>{
+	@Override
```

Comment:
Can you add a blank new line before this?

Reply:
i have fixed it